maitake-sync/src/wait_map.rs

```diff
@@ -490,6 +490,11 @@ impl<K: PartialEq, V> WaitMap<K, V> {
         }
     }
 
+    /// Returns `true` if this `WaitMap` is [closed](Self::close).
+    pub fn is_closed(&self) -> bool {
```

Comment:
similarly, it's nice to have a
```suggestion
    #[must_use]
    pub fn is_closed(&self) -> bool {
```